provision/acc_provision/acc_provision.py

```diff
@@ -946,9 +947,35 @@ def get_jinja_template(file):
     return template
 
 
-def generate_kube_yaml(config, output, operator_output, operator_cr_output):
-    template = get_jinja_template('aci-containers.yaml')
+def generate_operator_tar(tar_path, cont_docs, oper_docs):
 
+    # YAML file numbers generated start from 4 as first three are
+    # reserved for OpenShift specific files
+    file_start = 4
+    filenames = []
+
+    # Function to construct filenames for each yaml
+    def gen_file_list(docs, start, filenames):
+        for doc in docs:
+            filename = "cluster-network-" + str(start).zfill(2) + "-" + doc['kind'] + "-" + doc['metadata']['name'] + ".yaml"
+            filenames.append(os.path.basename(filename))
+            with open(filename, 'w') as outfile:
+                yaml.safe_dump(doc, outfile, default_flow_style=False, encoding="utf-8")
+            start += 1
+        return start
+
+    file_start = gen_file_list(cont_docs, file_start, filenames)
+    gen_file_list(oper_docs, file_start, filenames)
+
+    # Create tar for the parsed files and delete the files too
+    tar = tarfile.open(tar_path, "w:gz", encoding="utf-8")
+    for name in filenames:
+        tar.add(name)
+        os.remove(name)
```

Comment:
Perhaps create a new dir, and the tar all files in that dir to simplify some of the handling above (this can be a follow up PR).